src/test_progs/mod.rs

```diff
@@ -15,73 +15,76 @@
 
 /// Test programs for simplicity.
 pub(crate) mod hashblock;
-pub(crate) mod schnorr0;
-pub(crate) mod schnorr6;
-pub(crate) mod sighash_all;
+// FIXME: Schnorr and SIGHASH_ALL programs have left assertions whose children are not a hidden node
```

Comment:
I found a silly mistake on my part (_left and right hidden nodes being reversed)_. Schnorr and SIGHASH_ALL work and I will force push the fix.